<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxWidget</class>
 <widget class="QWidget" name="BoxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit box</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appstyle.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20">
            <item>
             <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
              <item row="2" column="2">
               <widget class="QSpinBox" name="gamblePercentSpin">
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="boxNameEdit"/>
              </item>
              <item row="5" column="2">
               <widget class="QSpinBox" name="minAmountSpin">
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="boxNameLabel">
                <property name="text">
                 <string>Box name:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QSpinBox" name="gamblePriceSpin">
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="gamblePriceLabel">
                <property name="text">
                 <string>Gamble price:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLineEdit" name="permissionEdit"/>
              </item>
              <item row="6" column="1">
               <widget class="QLabel" name="maxAmountLabel">
                <property name="text">
                 <string>Maximum amount:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QSpinBox" name="maxAmountSpin">
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="permissionLabel">
                <property name="text">
                 <string>Permission:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="minAmountLabel">
                <property name="text">
                 <string>Minimum amount:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="gamblePercentLabel">
                <property name="text">
                 <string>Gamble percent:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="tooltipEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="14,14,72">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="bulkAddItemButton">
         <property name="text">
          <string>Bulk add items</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bulkAddDinoButton">
         <property name="text">
          <string>Bulk add dinos</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Items</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>23</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="60,20,20">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="submitButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="appstyle.qrc"/>
 </resources>
 <connections/>
</ui>
